godsfapi/v3/cmd/examples/main.go

```diff
@@ -1,3 +1,4 @@
+// Deprecated: This package was deprected, please visit https://github.com/Duet3D/dsf-go.
```

Comment:
If you want to make it extra nice you could add the full path to the according version in the target URL, i.e. append `/v3` for `v3` and `/v2` for `v2` though that is technically not necessary.

Reply:
It is intended, it is referring to the github repository and not the go package.

Reply:
Yes, but you point from the existing `/v3` package back to `v1` this way. Just saying.

Reply:
not anymore. packages was moved into a subdirectory.